manifests/tftp.pp

```diff
@@ -29,4 +29,75 @@
       root => $foreman_proxy::tftp_root,
     }
   }
+
+  case $::osfamily {
+    'RedHat': {
+      case $::operatingsystem {
+        'Fedora': {
+          if versioncmp($::operatingsystemrelease, '19') >= 0 or $::operatingsystemrelease == 'Rawhide' {
```

Comment:
I don't think we support anything older than F19, or even F19 at all so it can simply be redhat. I'd simplify it to the following:


```puppet
case $::osfamily {
  'RedHat': {
    if versioncmp($::operatingsystemrelease, '7') >= 0 {
      $grub_type = 'redhat'
    } else {
      $grub_type = 'redhat_old'
    }
  },
}
```